core/pki/descriptor.go

```diff
@@ -152,11 +174,9 @@ func (d *MixDescriptor) UnmarshalBinary(data []byte) error {
 	if err != nil {
 		return err
 	}
-	_, err = cert.Verify(d.IdentityKey, data)
-	if err != nil {
-		return err
+	if sigs != nil && len(sigs) != 0 {
+		d.Signature = &sigs[0]
```

Comment:
len(sigs) > 1 used to return `ErrTooManySignatures` I think? now we we don't care?

Reply:
unmarshal binary should be used for cryptography. all wrong.

Reply:
i'm going to have to fix this bullshit later.